src/dodal/devices/xbpm_feedback.py

```diff
@@ -22,6 +26,29 @@ def __init__(self, prefix: str = "", name: str = "xbpm_feedback") -> None:
 
     @AsyncStatus.wrap
     async def trigger(self):
-        async for value in observe_value(self.pos_stable):
-            if value:
-                return
+        async def periodic_reminder():
+            schedule = [  # seconds, frequency
+                (0.1, 3),
```

Comment:
Should: I think the gap here is a bit small:
```suggestion
                (1, 3),
```